OsmIntegrator/Database/ApplicationDbContext.cs

```diff
@@ -28,21 +28,18 @@ public class ApplicationDbContext :
 
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
-            // Self many-to-many implemented thanks to this solution: 
-            // https://stackoverflow.com/questions/49214748/many-to-many-self-referencing-relationship
-            // Cascade delete was disabled: https://docs.microsoft.com/pl-pl/ef/core/saving/cascade-delete
             modelBuilder.Entity<DbConnections>()
                 .HasKey(t => new { t.Id });
 
             modelBuilder.Entity<DbConnections>()
                 .HasOne(c => c.OsmStop)
-                .WithMany(o => o.Connections)
+                .WithMany(o => o.OsmConnections)
                 .HasForeignKey(c => c.OsmStopId)
                 .OnDelete(DeleteBehavior.NoAction);
 
             modelBuilder.Entity<DbConnections>()
                 .HasOne(c => c.GtfsStop)
-                .WithMany()
+                .WithMany(o => o.GtfsConnections)
```

Comment:
To było niewystarczające. w momencie kiedy mieliśmy Stop.Connections było zawsze nulem w przypadku GtfStopów